### DISCUSSION TITLE
How do I create a seperate page for content items on Landing Page

### DISCUSSION THREAD

**Discussion by cerveser** [OrchardCMS/OrchardCore#7926] [Q&A] [answered]
Hi all, 
I'm using the agency theme. On the LandingPage the're content items called Services. Is it possible to also make a seperate Page out of these items? For example: I have one Service under Services. The service has an Autoroute, but when I use that path nothing is found.

**Comment by hishamco** [comment]
Is the intend to create a separate page for service (Service Details)?

**Comment by jtkech** [comment, marked as answer]
@cerveser 

The contained items in a `ListPart` are real content items in the database, but this is not the case for the contained items of a `BagPart`, and this is the case e.g. for the `Services` `BagPart` of the `LandingPage` and that embeds the `Service` items.

That said we support contained item routing, in the `AutoroutePart` settings of the `LandingPage` you need to check `Allow contained item routing`, and you need to add to e.g. the `Service` type the `AutoroutePart` and in its settings set `Manage contained item routes`

Then edit your landing page and check `Route contained items`, and specify the routes of your contained services or just publish the page and then see the auto generated routes.

E.g. you will be able to use this route `home-page/e-commerce`, the first segment `home-page` being the route of the container, here your landing page, and that you can change

**Reply by cerveser** [reply]
Thank you so much, I was playing around with this and I couldn't figure it out. So thanks again!
